src/inject/dynamic-theme/index.ts

```diff
@@ -133,10 +141,11 @@ function createStaticStyleOverrides() {
     const rootVarsStyle = createOrUpdateStyle('darkreader--root-vars');
     document.head.insertBefore(rootVarsStyle, variableStyle.nextSibling);
 
-    const proxyScript = createOrUpdateScript('darkreader--proxy');
-    proxyScript.append(`(${injectProxy})(!${fixes && fixes.disableStyleSheetsProxy})`);
+    const proxyScript = createOrUpdateScript('darkreader--proxy', 'inject/proxy.js', !(fixes && fixes.disableStyleSheetsProxy));
     document.head.insertBefore(proxyScript, rootVarsStyle.nextSibling);
-    proxyScript.remove();
```

Comment:
Calling `proxyScript.remove();` immediately might be technically a bit racy (it's not in practice). I moved it into `onload` handler to avoid the race. However, I would prefer to actually move it into the injected file `src/inject/dynamic-theme/stylesheet-proxy.ts` instead.

This will simplify the code as well.

Reply:
I fixed this already.